build/_build/Build.Steps.cs

```diff
@@ -254,6 +255,52 @@ partial class Build
             CopyFileToDirectory(source, dest, FileExistsPolicy.Overwrite);
         });
 
+    Target DownloadLibSqreen => _ => _
+        .Unlisted()
+        .After(CreateRequiredDirectories)
+        .OnlyWhenStatic(() => IsLinux)
+        .Executes(() =>
+        {
+            var wc = new WebClient();
+            var libSqreenUri = new Uri("https://www.nuget.org/api/v2/package/libsqreen/1.1.2.2");
+            var libSqreenZip = RootDirectory / "libsqreen.zip";
+            wc.DownloadFile(libSqreenUri, libSqreenZip);
+
+            Console.WriteLine($"{libSqreenZip} downloaded. Extracting to {LibSqreenDirectory}...");
+
+            UncompressZip(libSqreenZip, LibSqreenDirectory);
+        });
+
+    Target CopyLibSqreen => _ => _
+        .Unlisted()
+        .After(Clean)
+        .After(DownloadLibSqreen)
+        .OnlyWhenStatic(() => IsWin || LinuxArchitectureIdentifier != "arm64")
+        .Executes(() =>
+        {
+            if (IsWin)
+            {
+                foreach (var architecture in new[] {"win-x86", "win-x64"})
+                {
+                    var source = LibSqreenDirectory / "runtimes" / architecture / "native" / "Sqreen.dll";
+                    var dest = TracerHomeDirectory / architecture;
+                    Logger.Info($"Copying '{source}' to '{dest}'");
+                    CopyFileToDirectory(source, dest, FileExistsPolicy.Overwrite);
+                }
+            }
+            else
+            {
+                var (architecture, ext) = GetUnixArchitectureAndExtention();
+                var sqreenFileName = $"libSqreen.{ext}";
+
+                var source = LibSqreenDirectory / "runtimes" / architecture / "native" / sqreenFileName;
+                var dest = TracerHomeDirectory;
```

Comment:
For windows we write to `TracerHomeDirectory / architecture`, is that what you mean here too?
```suggestion
                var dest = TracerHomeDirectory / architecture;
```

Reply:
This is to tracer-home, my understanding is that this directory should contain the binaries in the structure they're need for the Unix installers. As we only support 64-bit under *nix OSs we place the libraries in a root. That why `CreateDdTracerHome` has actions to move them to architecture specific folders.

(Which means the changes in `CreateDdTracerHome` could be wrong, since we copy, rather than move, the sqreen library here).

Reply:
Yeah, it should mirror the tracer home, but I'd be strongly inclined to use the dd-trace layout nevertheless. Especially given we're considering switching to that in 2.0...

Reply:
I prefer not to mix the two layout styles, If we do migrate, I think it would be easier migrate both libraries to a different location, rather than trying to understand why the two libraries are stored in different location.